articles/aggregation-in-dbms/index.md

```diff
@@ -0,0 +1,78 @@
+* **Security:** Data sharing is done in the form of data encryptions, which ensure that there is no security threat. 
+* **Back up:** DBMS has a backup and recovery feature that enables it to restore the consistent state in the event of a system crash. 
+* **Data independence:** Any changes to data do not affect the application.
+* **Restricted Access:** A DBMS consists of defined roles, which enhances restricted access for specific data. This boosts the level of data security within the system. 
+
+### Aggregation in DBMS
+Aggregation refers to the process by which entities are combined to form a single meaningful entity. The specific entities are combined because they do not make sense on their own. To establish a single entity, aggregation creates a relationship that combines these entities. The resulting entity makes sense because it enables the system to function well. 
+
+When using data in the form of numerical values, the following operations can be used to perform DBMS aggregation;
+* **Average (AVG):** This function provides the mean or average of the data values. 
+* **Sum:** This provides a total value after the data values have been added. 
+* **Count:** This provides the number of records.
+* **Maximum (Max):** This function provides the maximum value of a given set of data.
+* **Minimum (Min):** This provides the minimum value of a given set of data. 
+* **Standard deviation (std dev):** This provides the dispersion or variation of the sets of data.
+
+### Reasons for Using Aggregation in DBMS
+Aggregation is used when the DBMS has the following characteristics.
+
+* **Many trivial entities:** A DBMS may consist of many entities that are not significant enough to provide meaningful information. In such a case, the trivial entities can be combined into one complex entity through aggregation.
+* **One trivial entity:** Aggregation is also needed if a DBMS has a single trivial entity that should be used for multiple operations. In this case, the trivial entity is used to form relationships with other entities. This may lead to many aggregation entities depending on the operations required.
+* **Inapplicable entity-model relationship:** The entity-model relationship cannot be applied to certain entities within the system. These specific entities can be combined with other entities to allow the application of the entity-model relationship in the entire system. This ensures that all the entities in the system are utilized. 
+
+### Process Flow for Aggregation in DBMS
+Aggregation in DBMS can be explained using [the entity-relationship model (ER model)](https://www.tutorialspoint.com/dbms/er_model_basic_concepts.htm). This is a conceptual diagram that represents the structure of a database and its components. It contains the relationships, attributes, and entities in a DBMS. This is similar to the columns, rows, and tables in a database. 
+
+The following are the main types of relationships in an ER model.

```

Comment:
Examples for each of these type of relationships would be a good addition.

Reply:
Done.